app/services/activities/activity_webhook_service.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Activities
+  class ActivityWebhookService
+    def initialize(webhook, activity)
+      @webhook = webhook
+      @activity = activity
+    end
+
+    def send_webhook
+      WebhookService.new(@webhook, activity_payload).send_webhook
+    end
+
+    def activity_payload
```

Comment:
not sure if this is correct or if we want to define the payload as something else, and if we maybe want to move it into a serializer or some other place